### DISCUSSION TITLE
Problems with Github education pack

### DISCUSSION THREAD

**Discussion by JadeBetty** [community/community#102824] [GitHub Education]
### Hi everyone,

I have been experiencing numerous of problems when trying to apply for Github education pack.
Sometimes, my school is under a different name, the school have changed their name.
Sometimes, my school is not listed on the available options on the maps.
If you don't have your school email, your application will take longer.
For me, my application was rejected because I didn't give them my school email? And my School doesn't provide me with my school email.
Its available for staff only. 
I have to take a picture of my receipt from my phone, upload it to google drive and then download it back to upload on the application process. Which I still got rejected for because "The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation." and my school logo contains the school name.
"Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy."
You can't use the back camera for taking pictures of your school document on mobile applications.
You have to use your front camera, which is bad.

**Comment by phuccoder2000** [comment]


**Reply by JadeBetty** [reply]
I have contacted Github support, will try and let you know.

**Reply by JadeBetty** [reply]
Ah great
Individual manual reviews and status updates are no longer provided.

**Reply by JadeBetty** [reply]
Not yet
I've to submit another application
